src/RunningMode/Webhook.php

```diff
@@ -61,18 +61,28 @@ public function processUpdates(Nutgram $bot): void
      */
     public function isSafe(): bool
     {
-        $ip_dec = (float)sprintf("%u", ip2long($_SERVER['REMOTE_ADDR']));
+        $ip = $this->ipToLong($_SERVER['REMOTE_ADDR']);
 
         foreach ($this->telegramIpRanges as $ipRange) {
             // Make sure the IP is valid.
-            if ($ip_dec >= (float)sprintf("%u", ip2long($ipRange['lower'])) &&
-                $ip_dec <= (float)sprintf("%u", ip2long($ipRange['upper']))) {
+            if ($ip >= $this->ipToLong($ipRange['lower']) &&
+                $ip <= $this->ipToLong($ipRange['upper'])) {
                 return true;
             }
         }
        return false;
     }
 
+
+    /**
+     * @param $ip
+     * @return float
+     */
+    private function ipToLong($ip): float
+    {
+        return (float)sprintf("%u", ip2long($ip));
```

Comment:
Why a long -> to string -> to float is done?

Reply:
To me looks totally unecessary, also the name of the function should be ipToFloat, given the return value
https://3v4l.org/iIngp#v8.0.15

Reply:
Yes ,actually this method is not necessary